src/App.test.js

```diff
@@ -33,3 +35,39 @@ test('food item is searched when button is clicked', () => {
   expect(screen.getByPlaceholderText('Search...')).toHaveValue('chicken');
   userEvent.type(input);
 });
+
+test('succesfully renders the login page button', () => {
+  render(<Login />);
+  const loginElement = screen.getByText('Login');
+  fireEvent.click(loginElement);
+});
+
+test('pass invalid password registration value to get error message', () => {
+  render(<Registration />);
+  const registerButton = screen.getByText('Register');
+  const username = screen.getByPlaceholderText('Username');
+  const password = screen.getByPlaceholderText('Password');
+
+  userEvent.type(username, 'Dragovich');
+  expect(screen.getByPlaceholderText('Username')).toHaveValue('Dragovich');
+  userEvent.type(password, '1');
+  expect(screen.getByPlaceholderText('Password')).toHaveValue('1');
+  expect(screen.queryByTestId('error-msg')).toBeInTheDocument();
+  expect(screen.queryByTestId('error-msg').textContent).toEqual('Use at least 4 characters');
+
+  fireEvent.click(registerButton);
+});
+
+test('pass valid registeration values in text fields to register', () => {
+  render(<Registration />);
+  const registerButton = screen.getByText('Register');
+  const username = screen.getByPlaceholderText('Username');
+  const password = screen.getByPlaceholderText('Password');
+
+  userEvent.type(username, 'Dragovich');
+  expect(screen.getByPlaceholderText('Username')).toHaveValue('Dragovich');
+
+  userEvent.type(password, '1234');
+  expect(screen.getByPlaceholderText('Password')).toHaveValue('1234');
+  fireEvent.click(registerButton);
+});
```

Comment:
This test looks good to me. Maybe add an `expect(registerButton).toBeInTheDocument()` call, since right now the click event is being called without asserting its existence.